### PR TITLE
Translate German Chapter 4 (part 2)

### PR DESCRIPTION
**Pull request by fabridamicelli**

### CODE REVIEW

**Comment by fabridamicelli** [comment]
@lewtun here's the 2nd part of chapter 4 ready to be merged (check out the PR docs [here](https://moon-ci-docs.huggingface.co/learn/nlp-course/pr_640/de/chapter4/6?fw=pt))

**Comment by fabridamicelli** [comment]
@lewtun any plans to merge this one? :)

**Comment by fabridamicelli** [comment]
Hi @mariagrandury @MKhalusova 
any plans to merge this PR?
Things move so fast that this information will soon be obsolete otherwise :)
Thanks in advance for taking a look!

File: chapters/de/chapter4/4.mdx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+Ausreichende Information zu der Vor- und Nachaufarbeitung der Daten dient auch dazu, dass man die Einschränkungen, Biases und den Kontext identifizieren kann, wann das Modell nützlich ist und wann nicht.
+
+Deswegen ist die Erstellung einer klar definierten Modellkarte ein sehr wichtiger Schritt. Hier geben wir ein Paar Hinweise, die dir dabei helfen könnten. Die Modellkarte wird durch eine *README.md* Datei (eine Markdown Datei) kreiert, die du schonmal gesehen hast.
+
+Das Konzept von Modellkarte ("model card") stammt aus einer Forschungsrichtung bei Google, die zuerst in dem Paper ["Model Cards for Model Reporting"](https://arxiv.org/abs/1810.03993) von Margaret Mitchell et al erschien. Vieles von dem, was hier steht, basiert auf dem Paper und wir empfehlen dir, das Paper zu lesen, um besser zu verstehen, warum Modellkarten so wichtig sind, wenn man Wert auf Reproduzierbarkeit, Wiederverwendbarkeit und Fairness legt.
+
+The model card usually starts with a very brief, high-level overview of what the model is for, followed by additional details in the following sections:
```
Comment (MKhalusova) [inline chapters/de/chapter4/4.mdx:18]: I think you can remove this original sentence.

**Review by MKhalusova — COMMENTED** [review]
Thanks for the translation! Looks good to me (though my German is very limited). Found a small nit, then we can merge.

File: chapters/de/chapter4/4.mdx (hunk 1 above)
Comment (fabridamicelli) [reply chapters/de/chapter4/4.mdx]: thanks for taking a look!

well spotted, you're right, that was a leftover and I just removed it :)

**Review by MKhalusova — APPROVED** [review]
LGTM, thank you!

**Pull request merged**